JS/Lesson30/goksel/assignment-files/job-board-app/src/components/Header/header.jsx

```diff
@@ -0,0 +1,11 @@
+import headerImage from "../../images/bg-header-desktop.svg";
+import "./header.css";
+
+export const Header = () => {
+
+    return (
+        <header>
+            <img src={headerImage} alt='header-image' />
```

Comment:
It's important to use meaningful text in the alt attribute, e.g. "Abstract background in different shades of green"

Reply:
Also, you use an image element here with a desktop size image and try to re-set it as a background of an image element on smaller screens. This is not right way of solving it. Instead, try using picture tag directly in the react component to load different images on different devices. 
https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images